<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tlinsStatusIcons</class>
 <widget class="QWidget" name="tlinsStatusIcons">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>46</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QCheckBox" name="isConnected">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Connected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Device status: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="calibrationStatus">
       <property name="text">
        <string>dddd</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="lineWidth">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="tlinsWDecimalValue" name="posX" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_3">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="tlinsWDecimalValue" name="posY" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>15</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_4">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>RA:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="tlinsWCoordinateHourAngle" name="posRA" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="Line" name="line_5">
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>2</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>DEC:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="tlinsWCoordinateDeclination" name="posDEC" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>25</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tlinsWDecimalValue</class>
   <extends>QWidget</extends>
   <header location="global">tlinswdecimalvalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlinsWCoordinateDeclination</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatedeclination.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tlinsWCoordinateHourAngle</class>
   <extends>QWidget</extends>
   <header>tlinswcoordinatehourangle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
